.github/workflows/docker.yml

```diff
@@ -66,8 +66,7 @@ jobs:
       parent-image: ""
       image: docker-stacks-foundation
       platform: aarch64
-      runs-on: ARM64_FAST
-    if: github.repository_owner == 'jupyter'
+      runs-on: ubuntu-24.04-arm
```

Comment:
Github rep has said that they believe the ubuntu-22.04-arm runners may be stable @mathbunnyru - I think using ubuntu 22.04 as runner environment may be fine still - I'm not fully sure.

Reply:
> Github rep has said that they believe the ubuntu-22.04-arm runners may be stable @mathbunnyru

Great, I'll try it out!

> I think using ubuntu 22.04 as runner environment may be fine still - I'm not fully sure.

Definitely still fine, because that's what self-hosted runners are still running, and I had 0 issues due to that 😄 
So, in a way, switching to `ubuntu-22.04-arm` is even better (we're changing less important things).

Reply:
Ooooh looking better so far - not seen all tests pass but some

Reply:
> Ooooh looking better so far - not seen all tests pass but some

Yes, and actually, the last run of `24.04` was fine as well.
But I'm completely  ok with switching to `ubuntu-22.04-arm` first